code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -130,8 +130,8 @@
 	sort_category = "Xenowear"
 	
 /datum/gear/suit/tajaran_labcoat
-	display_name = "people's republic medical coat (Tajara)"
+	display_name = "peoples republic medical coat (Tajara)"
```

Comment:
It is related to the bug. You can't select them in the loadout if the display name has an apostrophe.